### ISSUE TITLE
Procedural Flipping Reloaded

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#331]
Once again I want to touch on Procedural Flipping.To keep it simple, I want to work with Quad only. Logically, I will just need the Normal of each face. Then I need edge or some kind of orientation so that each face can flip nicely.

I started with this, but noticed something weird already. Is below supposed to happen?
![prtscr capture_11](https://cloud.githubusercontent.com/assets/3952687/3564402/790e3174-0a83-11e4-8d95-1e743f1c5745.png)

I think, I must have forgotten something important.

I tried something different, but then, wait this also does not look right.
![prtscr capture_12](https://cloud.githubusercontent.com/assets/3952687/3564408/d2e60d34-0a83-11e4-837e-9181a579d80b.png)

Basically for now, before I can "flip", I want to be able to scale each face from origin. 

I will have to look up my previous notes, I think I have this covered before, or I left it unanswered. Supposedly easier to do today.

BLEND:
https://www.dropbox.com/s/oeonq1hijru8o3c/random_procedural_flipping_002.blend
